examples/decoding/plot_decoding_spatio_temporal_source.py

```diff
@@ -44,7 +49,7 @@
 
 # Setup for reading the raw data
 raw = io.read_raw_fif(raw_fname, preload=True)
-raw.filter(2, None, fir_design='firwin')  # replace baselining with high-pass
+raw.filter(0.1, None, fir_design='firwin')
```

Comment:
never mind my last comment. But the acquisition highpass is 0.1, no? What's the benefit of doing it again?